tests/fixtures.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+"""
+
+    tests.fixtures
+    ~~~~~~~~~~~
```

Comment:
underline as long as the text (maybe a font problem here on gh)

Reply:
Not a font problem, I copied the header from tests.utils and forgot to extend the underline. I'll fix that.